eve/main.yml

```diff
@@ -1658,6 +1659,193 @@ stages:
           <<: *terraform_destroy
           workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+
+  bootstrap-restore:
+    _metalk8s_internal_info:
+      junit_info: &_bootstrap_restore_junit_info
+        TEST_SUITE: install
+        CLASS_NAME: multi node.centos7
+        TEST_NAME: bootstrap restore
+      simultaneous_builds: 20
+    worker: *multi_node_worker
+    steps:
+      - Git: *git_pull
+      - ShellCommand:
+          <<: *add_final_status_artifact_failed
+          env:
+            <<: *_env_final_status_artifact_failed
+            <<: *_bootstrap_restore_junit_info
+            STEP_NAME: bootstrap-restore
+      - ShellCommand: *setup_cache
+      - ShellCommand: *ssh_ip_setup
+      - ShellCommand: *retrieve_iso
+      - ShellCommand: *retrieve_iso_checksum
+      - ShellCommand: *check_iso_checksum
+      - ShellCommand: *terraform_install
+      - ShellCommand: *terraform_install_check
+      - ShellCommand: *terraform_init_multi_node
+      - ShellCommand: *terraform_validate_multi_node
+      - ShellCommand:
+          <<: *terraform_apply_multi_node
+          env: &_env_terraform_bootstrap_restore
+            <<: *_env_terraform
+            TF_VAR_nodes_count: "2"
+      - ShellCommand: *check_ssh_config_bootstrap
+      - ShellCommand:
+          <<: *copy_bastion_pub_key_ssh
+          env:
+            HOSTS_LIST: "bootstrap node1 node2"
+      - ShellCommand: *copy_bastion_priv_key_to_bootstrap
+      - ShellCommand: *copy_iso_to_bootstrap
+      - ShellCommand: *create_mountpoint_on_bootstrap
+      - ShellCommand: *mount_iso_on_bootstrap
+      - ShellCommand: *bootstrap_on_bootstrap
+      - ShellCommand: *enable_ipip_ssh
+      - ShellCommand: *copy_test_source_to_bastion
+      - ShellCommand: *wait_kube_system_pods_running_multi_node
+      - ShellCommand:
+          <<: *bastion_tests
+          name: Run installation scenario on the bastion
+          env:
+            <<: *_env_bastion_tests
+            PYTEST_FILTERS: "install and ci and multinodes"
+      - ShellCommand: *provision_volumes_on_node1
+      - ShellCommand: *wait_pods_running_multi_node
+      - ShellCommand: *multi_node_fast_tests
+      - ShellCommand: *multi_node_slow_tests
+      - SetPropertyFromCommand:
+          name: Set bootstrap backup archive property
+          property: bootstrap_backup_archive
+          command: >
+            ssh -F ssh_config bootstrap
+            "sudo find /var/lib/metalk8s -name 'backup_*.tar.gz'
+            -printf '%f\n' | sort | tail -n1"
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Retrieve the backup archive from the bootstrap node
+          command: >
+            scp -F ssh_config
+            bootstrap:"/var/lib/metalk8s/%(prop:bootstrap_backup_archive)s"
+            /tmp
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
```

Comment:
Would have been simpler in practice to just scp from Bootstrap to Node 2 directly ;)

Reply:
Why ? We create a new node for bootstrap we do not use the node 2

Reply:
Ah right my bad.